packages/bpk-scrim-utils/readme.md

```diff
@@ -0,0 +1,32 @@
+# bpk-scrim-utils
+
+> Higher order component that adds a scrim behind components and manages scroll states
+
+## Installation
+
+```sh
+npm install bpk-scrim-utils --save-dev
+```
+
+```js 
+import { withScrim } from 'bpk-scrim-utils';
+
+const Box = props => (
+  <div 
+    ref={props.getDialogRef}
+    {...props.closeEvents}
+  >
+    <BpkButton onClick={props.onClose}>Close</BpkButton>
+    Hello
+  </div>
+);
+
+const BoxWithScrim = withScrim(Box);
+```
+
```

Comment:
I think there needs to be more docs here detailing the props that the HOC exposes to the wrapped component. `isIphone` is missing for example.

Reply:
done